examples/tfengine/multi_envs.hcl

```diff
@@ -120,13 +120,21 @@ template "cicd" {
     build_editors = ["group:example-cicd-editors@example.com"]
 
     terraform_root = "terraform"
+
+    # IMPORTANT: Cloud Source Repositories does not support code review or
+    #   presubmit runs. If we set both plan and apply to run at the same time,
+    #   they will conflict when trying to acquire the lock and may error out.
+    #   To get around this, below we only auto-apply for non-prod envs,
+    #   and only plan for the prod env.
     envs = [
       {
         name        = "shared"
         branch_name = "shared"
         triggers = {
           validate = {}
-          plan = {}
+          plan = {
```

Comment:
we can just simply remove this plan block

Reply:
I think it would still be good to keep the job there, it can be useful to run manually, i.e. for debugging.

Reply:
actually for `shared`, we are treating it as prod.  So only remove for dev